src/Behat/Behat/Output/Node/Printer/Progress/ProgressStepPrinter.php

```diff
@@ -52,6 +56,10 @@ public function printStep(Formatter $formatter, Scenario $scenario, StepNode $st
         $printer = $formatter->getOutputPrinter();
         $style = $this->resultConverter->convertResultToString($result);
 
+        if ($this->hasPrintedOutput) {
```

Comment:
After printing any output, we need to print a new line before continuing to print the "dots" of the progress

Reply:
I would suggest perhaps putting this as a comment in the code for future reference?

Reply:
Also, I might be missing it but how does this flag get set back to false if the next step does *not* have any output?

Reply:
Comment added. Regarding setting the flag back to false, I made a last minute change to the code and when doing so, I accidentally removed this. Thanks for noticing it!